acceptance/tests/sync/sync_catalog_test.go

```diff
@@ -65,7 +65,7 @@ func TestSyncCatalog(t *testing.T) {
 
 			service, _, err := consulClient.Catalog().Service(syncedServiceName, "", nil)
 			require.NoError(t, err)
-			require.Equal(t, 1, len(service))
+			require.Len(t, service, 1)
```

Comment:
Would it make sense to add assertions here on the service's healthcheck, maybe just that its healthy to confirm it got synced?

Reply:
I tried to add an assertion here but a Catalog Service did not contain health check info 😞

Reply:
Like a catalog service has HealthChecks but not AgentChecks which is what we configure which is a part of the catalog registration

Reply:
i added an assertion based on how endpoints controller tests healthchecks. 🤞